clion-plugin/src/main/kotlin/org/utbot/cpp/clion/plugin/ui/wizard/UTBotWizard.kt

```diff
@@ -12,10 +12,12 @@ import org.utbot.cpp.clion.plugin.ui.wizard.steps.BuildOptionsStep
 import org.utbot.cpp.clion.plugin.ui.wizard.steps.ConnectionStep
 import org.utbot.cpp.clion.plugin.ui.wizard.steps.IntroStep
 import org.utbot.cpp.clion.plugin.ui.wizard.steps.FinalStep
+import org.utbot.cpp.clion.plugin.utils.getCurrentClient
+import java.awt.event.KeyEvent
 
 class UTBotWizard(private val project: Project) : AbstractWizard<UTBotBaseWizardStep>("UTBot: Quickstart", project) {
     // copy of settings to make changes during wizard steps
-    private val mySettingsModel = UTBotSettingsModel(project.settings.storedSettings, projectIndependentSettings)
+    private val mySettingsModel = UTBotSettingsModel(project.settings.storedSettings.copy(), projectIndependentSettings.copy())
```

Comment:
It is very unclear why should we call copy here. It seems that we require a copy of the whole setting model (that may call copying field bty field in its internals).
As far as I remember, we had to layers - settings themself and a model of settings that can be read, copied, modified...

Reply:
`project.settings.storedSettings` is a reference to State instance. Changes made through this reference during the wizard will affect the settings even if user  haven't finished the wizard. We need completely independent object to not make changes to settings, that's why copy() is used